lib/node.js

```diff
@@ -1036,7 +1065,7 @@ class Node {
       PARAMETER_EVENT_MSG_TYPE
     ))();
 
-    const secondsAndNanos = new Clock.ROSClock().now().secondsAndNanoseconds;
```

Comment:
qq. Since `new ROSClock()` calls malloc internally, wouldn't this be a memory leak since the clock has no parent handle and `release` is never called?

Reply:
If the `handle` doesn't have a parent, then the dtor will be called when this js object is GCed by v8, so the functional callback passed when creating this handle is called in `Reset`.
https://github.com/RobotWebTools/rclnodejs/blob/d3d5c2fe1ab4773121b6827076711ccebb159230/src/rcl_handle.cpp#L26-L28

Reply:
oh cool. I didn't know that. thanks